### DISCUSSION TITLE
Access current state of edit data and pass into relation manager

### DISCUSSION THREAD

**Discussion by alvinmr** [filamentphp/filament#6887] [Help] [answered]
![img](https://i.ibb.co/0C4Wbvb/image.png)

can I get the geojson field data that on parent to pass the data into relation manager?

**Comment by pboivin** [comment]
How do you want to use the field data? In a column, an action, something else?

**Reply by alvinmr** [reply]
I want to use the geojson field in the parent, to input of relation manager of custom field map input to get the geojson
![img](https://i.ibb.co/1GpVTcW/image.png)

**Reply by pboivin** [reply, marked as answer]
I think you're looking for `ownerRecord` :
https://filamentphp.com/docs/2.x/admin/resources/relation-managers#accessing-the-owner-record

**Reply by alvinmr** [reply]
Wow thanks, you really saved my day! didn't know it was here before 🤯
